backend/src/meme/model/dto/meme.dto.ts

```diff
@@ -0,0 +1,15 @@
+import {SourceType} from "../meme.enums";
+
+export class MemeDto {
+    constructor(name: string, sourceType: SourceType, source: string, extension: string) {
```

Comment:
Why did u use constructor here?

Use is as a object where u can validate STM, ep. :

```ts
export class UpdateUserDto {
  @Length(5, 30)
  @IsAlphanumeric()
  @IsString()
  username: string;
}
```

And create entity object, use it only as type for argument object.

ep. :

```ts
 async create(userCredentialsDto: UserCredentialsDto)
 ```
 
 IMO that's good way

Reply:
I use this constructor in `addMemeByFile`